dd-java-agent/instrumentation/javax-mail/build.gradle

```
@@ -0,0 +1,17 @@
+muzzle {
+  pass {
+    coreJdk()
+  }
+}
+
+apply from: "$rootDir/gradle/java.gradle"
+apply plugin: 'call-site-instrumentation'
+
+addTestSuiteForDir('latestDepTest', 'test')
+
+dependencies {
+  testRuntimeOnly project(':dd-java-agent:instrumentation:iast-instrumenter')
+  implementation 'javax.mail:javax.mail-api:1.4.4'
```

```suggestion
  compileOnly 'javax.mail:javax.mail-api:1.4.4'
```

We do not really want to include javax.mail in our tracer.

---

If it doesn't find classes for tests, just also add:
`testImplementation 'javax.mail:javax.mail-api:1.4.4'`
(in addition to the `compileOnly` one)

---

@manuel-alvarez-alvarez 😵

---

actually no, this isn’t the smoke test. it makes sense